drop "this file is part of systemd" and lennart's copyright from header

poettering opened the pull request:
These headers are useless these days and more often incorrect than not. Let's go on a useless header diet, and drop "this file is part of systemd" as well as lennart's copyright lines from the header. Let's make SPDX and git history the one true source of information for all of this. it's more finegrained, generally more accurate and carries full history. 

This doesn't drop anyone else's copyright lines, just my own as that might be understood as an unfriendly act.

mbiebl commented:
For files which are installed into the system (say pam config files, hwdb files etc), I find it somewhat useful if those files contain a "# This file is part of systemd." header.

For source files, dropping that header seems reasonable indeed though.

keszybz commented on man/bootctl.xml:
```diff
@@ -4,10 +4,9 @@
-  This file is part of systemd.
 -->
 
+
```
Not needed, I think.

poettering commented:
> Did you use some script or magic invocation? If you could paste is somewhere, I'd drop my copyrights too.

I used an awfully hacky command line: `sed -i -e '/Copyright [0-9][0-9][0-9][0-9] Lennart Poettering/d' src/*/*.[ch] src/*/*/*.[ch] man/*.xml`.

poettering commented:
OK, force pushed a new version. I changed to leave any files that are installed with their "This file is part of" line. I also fixed the other issue @keszybz pointed out. PTAL!

keszybz commented:
First patch:
```console
$ git show --stat 206f9d6a15|grep '\.conf.[^x]'
 src/core/org.freedesktop.systemd1.conf                | 2 --
 src/core/system.conf.in                               | 2 --
 src/core/user.conf                                    | 2 --
 src/coredump/coredump.conf                            | 2 --
 src/hostname/org.freedesktop.hostname1.conf           | 2 --
 src/import/org.freedesktop.import1.conf               | 2 --
 src/journal/journald.conf                             | 2 --
 src/locale/org.freedesktop.locale1.conf               | 2 --
 src/login/logind.conf.in                              | 2 --
 src/login/org.freedesktop.login1.conf                 | 2 --
 src/machine/org.freedesktop.machine1.conf             | 2 --
 src/network/org.freedesktop.network1.conf             | 2 --
 src/resolve/org.freedesktop.resolve1.conf             | 2 --
 src/resolve/resolved.conf.in                          | 2 --
 src/timedate/org.freedesktop.timedate1.conf           | 2 --
 src/timesync/org.freedesktop.timesync1.conf           | 2 --
 src/timesync/timesyncd.conf.in                        | 2 --
 units/user-.slice.d/10-defaults.conf                  | 2 --
```
Those are installed files.

poettering commented:
not sure how i missed those

poettering commented:
OK, force pushed another version, restoring the line for those .conf files too. no other changes. PTAL!
